docs/docs/reference/data_preparation/data_filtering.md

```diff
@@ -41,7 +42,44 @@ This macro allows you remove duplicate rows from a dbt model based on certain co
  2   |  completed   |    13:05:23     |
  3   |  completed   |    13:30:00     |
 ```
+### [filter_get_duplicates](https://re-data.github.io/dbt-re-data/#!/macro/macro.re_data.filter_get_duplicates)
+*Arguments:*
+- *relation: dbt base model to perform the filtering on*
+- *unique_cols: List of columns that uniquely identify each row*
+- *sort_columns: Order in which we want to sort the partitioned rows. e.g. (created_at DESC, created_at ASC to choose the latest or earliest row based on the timestamp column*
+
+Return type: table with duplicate rows
+
+along with the fields of the base model the macro returns duplication context in new fields:
+re_data_duplicates_count - total number of duplicates with the same current key set
+re_data_duplicate_row_number - number of current duplicate row inside the group of duplicates with the same current key set
+
 
+This macro allows you to get duplicate rows from a dbt model based on certain conditions.
+
+```sql
+  id |  status      |   updated_at    |
+--------------------------------------+
+ 1   |  pending     |    13:00:45     |
+ 2   |  completed   |    13:05:23     |
+ 1   |  completed   |    13:10:35     |
+ 2   |  pending     |    13:04:49     |
+ 3   |  completed   |    13:30:00     |
+
+ => SELECT id, status, updated_at,
```

Comment:
as rest docs are using small letters can you make this SELECT/WHERE small?

Reply:
done